### PR TITLE
Rspec testing

### PR DESCRIPTION
**Pull request by MiguelDP4**
- Created specs for all enumerable methods.
- Tested several cases for each enumerable method.

### CODE REVIEW

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @bren2102 and @MiguelDP4 :balloon: :lollipop: 

### Your test cases are very well implemented, I also reviewed your tic-tac-toe test and the changes to make here are the same as I requested for your tic-tac-toe

## Project specific

**let use**

- [ ] you should use [let](https://github.com/rubocop-hq/rspec-style-guide#let-blocks) to define variables within your test cases so use it instead of defining this multiple times:

array = [1, 2, 5, 8, 7, 98, 4]

**negative test cases**

- [ ] add negative test cases (use not_to) to all your test cases for example:

```
.
.
let(:array) { [1, 2, 5, 8, 7, 98, 4] }
.
.
  describe '#select' do
    it 'Returns the filtered array if a block is given' do
      expect(array.select { |element| element > 1}).to eql([2, 5, 8, 7, 98, 4])
    end
    it 'doesn\'t return the unfiltered array if a block is given' do
      expect(array.select { |element| element > 1}).not_to eql(array)
    end
  end
.
.
```
### That's it, team! :raised_hands: 

### Please :pray:  do the changes :woman_technologist: :man_technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any questions? let me know on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

File: spec/enumerables_spec.rb [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    end
+
+    it 'Gets a symbol and uses it to operate' do
+      expect(array.inject { |accumulator, value| accumulator + value }).to eql(125)
+    end
+
+    it 'Gets a symbol and uses it to operate (negative)' do
+      expect(array.inject { |accumulator, value| accumulator + value }).not_to eql(200)
+    end
+  end
```
Comment (shshamim63) [inline spec/enumerables_spec.rb]: You were suppose to test my_inject. But I can see inject. So replace inject with my_inject.

**Review by shshamim63 — COMMENTED** [review]
## Status :space_invader: Changes required :space_invader:  
___
Hi @MiguelDP4 and @bren2102 :wave: ,
Great job :clap: , You have resolved the issues mentioned by the previous reviewer. But I have found a new issue.
___
### :spades: Mandatory Changes :spades: 
___
1. You have to check my_inject. But I can not see it. See my comment to resolve it.
___
#### Please resolve the issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Review by whiz25 — APPROVED** [review]
## Awesome :clap: 
You have made all the requested changes and therefore it's a :+1: from me. Merge and proceed.

### Martin Afani

**Pull request merged**